### DISCUSSION TITLE
LO/TO splitting in SCPH calculation

### DISCUSSION THREAD

**Discussion by Seonghyeonkweon** [ttadano/alamode#97] [Q&A]
First of all, thank you for providing a good phonon calculation system.
I am asking because of an unresolved problem during perovskite scph phonon calculation. (The model is BaTiO3.)

We aim to calculate the Curie temperature after obtaining each frequency value by performing LO and TO spliting through SCPH calculation.

We have confirmed that there are countless eigenvalues values in scph_band .
However, it is difficult to distinguish between LO and TO values among these many values.
Do you have any idea how to tell LO/TO apart?
![image](https://user-images.githubusercontent.com/115207966/194475710-add0e88c-ae0d-459f-bde5-6156ce85dd34.png)

**Comment by ttadano** [comment]
If you compare phonon dispersion curves obtained using  `NONANALLYTIC=0` and `NONANALYTIC=3`, you will see differences in the frequencies of LO phonons.

Another way is to compute the dot products (and cross products) of the phonon polarization vector and the momentum vector. If a phonon mode is transverse, you will have a large cross-product value and a small dot product. For a longitudinal mode, the magnitude of the dot product is larger.

**Reply by Seonghyeonkweon** [reply]
Thank you for reply ttadano.

I have some question, to there

I think it seems that command to extract the eigenvector is only available in mode (phonon). 
Is there a way to output eigenvector when calculate with mode (scph)?

And If there is a way, what output file can I extract from?

Thank you
